pkgs/applications/editors/emacs/elisp-packages/manual-packages/ebuild-mode/default.nix

```diff
@@ -1,22 +1,28 @@
-{ lib, melpaBuild, fetchurl, writeText }:
+{
+  lib,
+  melpaBuild,
+  fetchgit,
+  writeText,
+}:
 
 melpaBuild rec {
   pname = "ebuild-mode";
-  version = "1.67";
+  version = "1.70";
 
-  src = fetchurl {
-    url = "https://dev.gentoo.org/~ulm/emacs/ebuild-mode-${version}.tar.xz";
-    hash = "sha256-5qxHpu1BLtI8LFnL/sAoqmo80zeyElxIdFtAsfMefUE=";
+  src = fetchgit {
```

Comment:
We could do this with `fetchzip` to avoid a full clone: https://gitweb.gentoo.org/proj/ebuild-mode.git/snapshot/ebuild-mode-1.70.tar.bz2